docs/guides/applications/configuration-management/ansible/using-the-linode-ansible-collection-to-deploy-infrastructure/index.md

```diff
@@ -1,143 +1,180 @@
 {{</ caution >}}
 
 ## Before You Begin
 
 {{< note >}}
-The steps outlined in this guide require [Ansible version 2.9.10 or greater](https://github.com/ansible/ansible/releases/tag/v2.9.10), and were created using Ubuntu 22.04.
+The steps outlined in this guide require [Ansible version 2.9.10 or greater](https://github.com/ansible/ansible/releases/tag/v2.9.10) and were tested on a Linode running Ubuntu 22.04. The instructions can be adapted to other Linux distributions or operating systems.
 {{</ note >}}
 
--   Add a limited user to your Linode following the steps below, created by following the [Add a limited User Account](/docs/guides/set-up-and-secure/#add-a-limited-user-account) section of our  [Setting Up and Securing a Compute Instance](/docs/guides/set-up-and-secure/) guide. Ensure that all commands are entered as your limited user.
+1.  Provision a server that acts as the Ansible [*control node*](docs/guides/getting-started-with-ansible/#what-is-ansible), from which other compute instances are deployed. Follow the instructions in our [Creating a Compute Instance](https://www.linode.com/docs/guides/creating-a-compute-instance/) guide to create a Linode running Ubuntu 22.04. A shared CPU 1GB Nanode is suitable. You can also use an existing workstation or laptop if you prefer.
 
-- Ensure that you have performed system updates with the following command:
+1.  Add a limited Linux user to your control node Linode by following the [Add a Limited User Account](/docs/guides/set-up-and-secure/#add-a-limited-user-account) section of our [Setting Up and Securing a Compute Instance](/docs/guides/set-up-and-secure/) guide. Ensure that all commands for the rest of this guide are entered as your limited user.
 
-        apt update && apt upgrade
+1.  Ensure that you have performed system updates:
 
--   Install Ansible on your computer. Use the steps in the [Control Node Setup](/docs/guides/getting-started-with-ansible/#set-up-the-control-node) section of the [Getting Started With Ansible - Basic Installation and Setup](/docs/guides/getting-started-with-ansible/) guide.
+        sudo apt update && sudo apt upgrade
 
--   Ensure you have Python version 2.7 or higher installed on your computer. Issue the following command to check your system's Python version:
+1.  Install Ansible on your control node. Follow the steps in the [Install Ansible](/docs/guides/getting-started-with-ansible/#install-ansible) section of the [Getting Started With Ansible - Basic Installation and Setup](/docs/guides/getting-started-with-ansible/) guide.
+
+1.  Ensure you have Python version 2.7 or higher installed on your control node. Issue the following command to check your system's Python version:
 
         python --version
 
--   Install the official [Python library for the Linode API v4](https://github.com/linode/linode_api4-python).
```

Comment:
the Linode API python module automatically gets installed from the requirements.txt file of the Linode Ansible collection, so I removed this step